src/Mapper/ProgrammesDbToDomain/NetworkMapper.php

```diff
@@ -18,6 +18,8 @@ public function getCacheKey(array $dbNetwork): string
         return $this->buildCacheKey($dbNetwork, 'id', [
             'image' => 'Image',
             'defaultService' => 'Service',
+        ], [
+            'services' => 'Service',
```

Comment:
Services instead of Service?

Reply:
This is correct, anything else will not get the right cache key

Reply:
The key of this array corresponds to the property name in the Data Entity. The value is the name of the Mapper that knows how to build this item - in this case the `ServiceMapper`.